client/src/App.js

```diff
@@ -1,26 +1,32 @@
-import { useState } from 'react';
-import './App.css';
+import TasteCard from './components/TasteCard';
+import { tastes } from './database';
+import styled from 'styled-components';
 
 function App() {
-  const [users, setUsers] = useState([]);
-
-  function fetchSomething() {
-    fetch('/api')
-      .then(res => res.json())
-      .then(data => setUsers(data));
-  }
-
   return (
-    <div className="App">
-      <h1>Users</h1>
-      <button onClick={fetchSomething}>Fetch users</button>
-      <ul>
-        {users.map(user => (
-          <li key={user.id}>{user.name}</li>
+    <AppContainer>
+      <h1>Coffee Roast Levels</h1>
+      <CardWrapper>
+        {tastes.map((taste, id) => (
```

Comment:
Just to let you know: the `id` here is the `index` of the `taste`object in the `tastes` array. It is not a good practice to use the `index` as a `key` prop. I recommend using the `id` property from the `taste` onbject itself.